physical/raft/transaction.go

```diff
@@ -738,19 +779,22 @@ func (t *RaftTransaction) Rollback(ctx context.Context) error {
 		return physical.ErrTransactionAlreadyCommitted
 	}
 
+	t.cleanup.Stop()
+
 	// The transaction is done; release the permit pool entry when we're done
 	// here.
 	//
 	// Also unlock the read lock on the underlying fsm.
 	defer func() {
 		if t.writable {
-			lowestActiveIndex := t.b.fsm.fastTxnTracker.lowestActiveIndexAfterCommit(t.index)
+			t.b.fsm.fastTxnTracker.completeTransaction(t.index)
+			lowestActiveIndex := t.b.fsm.fastTxnTracker.lowestActiveIndex()
```

Comment:
`lowestActiveIndex()` was added after this code was written. I took the chance to clean this up.